src/main/kotlin/com/neva/gradle/fork/config/properties/PropertyDefinition.kt

```diff
@@ -16,11 +19,9 @@ open class PropertyDefinition @Inject constructor(val name: String) {
 
   var required = true
 
-  var validator: PropertyValidator.() -> Unit = {
-    if (required) {
-      required()
-    }
-  }
+  var controller: Property.() -> Unit = {}
+
+  var validator: PropertyValidator.() -> Unit = { notBlank() }
```

Comment:
Having field optional when I will only override a validator seems to me unpleasant surprise

Reply:
nope, this is fixed in `Property` / recovered to your version